src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import { node, string, shape } from 'prop-types';
+import Helmet from 'react-helmet';
+import HeaderContainer from '../containers/Header';
+import FooterContainer from '../containers/Footer';
+import { ServiceContextProvider } from '../contexts/ServiceContext';
+import { RequestContextProvider } from '../contexts/RequestContext';
+import GlobalStyle from '../lib/globalStyles';
+import ConsentBanner from '../containers/ConsentBanner';
+
+const PageWrapper = ({ bbcOrigin, children, data }) => {
+  const { isAmp, service } = data;
+  return (
+    <Fragment>
+      <GlobalStyle />
+      <RequestContextProvider
+        platform={isAmp ? 'amp' : 'canonical'}
+        bbcOrigin={bbcOrigin}
+      >
+        <ServiceContextProvider service={service}>
+          <Helmet>
```

Comment:
This should probably go in the article container

Reply:
Nop, ignore me, its fine here for now, eventually it will have a different, general, path

Reply:
Service context provider will be needed across pageTypes and help needs the values from it.

Reply:
haha ignored :p

Reply:
I was pointing at the helmet, for the manifest json, but nvm :P

Reply:
Yup, but helmet requires service, which it gets from the service context.